### PR TITLE
Update 01-25-2022 release notes

### PR DESCRIPTION
**Pull request by PaulClark2**
Update 12-28-2021 and 01-25-2022 release notes (16.6 and 16.innovation).

This only needs one review.

### CODE REVIEW

File: release_notes/release_notes.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
+# January 25, 2022 (16.Innovation)
+## Added
+
+- **Help for candidates and committees:** Added [dates and deadlines](https://www.fec.gov/help-candidates-and-committees/dates-and-deadlines/]) pages for 2022.
+
+## Changed
+
+- **Help for candidates and committees:** Updated administrative fine information and  links to FEC Record articles throughout [dates and deadlines](https://www.fec.gov/help-candidates-and-committees/dates-and-deadlines/]) pages
```
Comment (patphongs) [inline release_notes/release_notes.md:8]: Please remove the extra bracket at the end of this link

File: release_notes/release_notes.md [hunk 2]
```diff
@@ -1,3 +1,37 @@
+# January 25, 2022 (16.Innovation)
+## Added
+
+- **Help for candidates and committees:** Added [dates and deadlines](https://www.fec.gov/help-candidates-and-committees/dates-and-deadlines/]) pages for 2022.
```
Comment (patphongs) [inline release_notes/release_notes.md:4]: Also remove the extra bracket at the end of this link

**Review by patphongs — CHANGES_REQUESTED** [review]
@PaulClark2 Just two minor fixes before I can approve and merge, thanks!

File: release_notes/release_notes.md (hunk 2 above)
Comment (PaulClark2) [reply release_notes/release_notes.md]: Thanks @patphongs. I'm made the requested change.

File: release_notes/release_notes.md (hunk 1 above)
Comment (PaulClark2) [reply release_notes/release_notes.md]: Thanks @patphongs. I'm made the requested change.

**Review by patphongs — APPROVED** [review]
Thanks @PaulClark2, looks good!

**Pull request merged**